src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/ViewAllMoveTypesTest.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.integration
+
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.ChooseSupplier
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.Dashboard
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.Login
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.MoveDetails
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.MovesByType
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.Pages.SelectMonthYear
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.cancelledMoveM60
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.lockoutMoveM40
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.longHaulMoveM30
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.multiMoveM50
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.redirectMoveM20
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.standardMoveM4
+import uk.gov.justice.digital.hmpps.pecs.jpc.move.Move
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Supplier
+import java.time.LocalDate
+import java.time.Month
+import java.time.Year
+
+internal class ViewAllMoveTypesTest : IntegrationTest() {
+
+  @Test
+  fun `go to a previous month and view all available move types`() {
```

Comment:
At the moment we only have test data for previous months.

I need to put some thought into generating data for the current month (this is not that straight forwards).  Also this will (should) be covered in the next integration test.